File: Codigo/Classes/Evento.m
classdef Evento
%   Estrutura de dado que possui informações dos eventos do EEG
    properties
        inicio        double
        fim           double
        tipo          string
        probabilidade double
    end

    methods
        % construtor da classe
        function this = Evento(inicio, fim, tipo, probabilidade)
            if (nargin == 4)
                this.inicio = inicio;
                this.fim = fim;
                this.tipo = tipo;
                this.probabilidade = probabilidade;
            end
        end
    end

end

File: Python/Classes/Evento.py
class Evento(object):

    inicio: float
    fim: float
    tipo: str
    probabilidade: float

    def __init__(self, inicio=0, fim=0, tipo="null", probabilidade=0):
        self.inicio = inicio
        self.fim = fim
        self.tipo = tipo
        self.probabilidade = probabilidade
